tests/Unit/NumericalAlgorithms/LinearAlgebra/Test_FindGeneralizedEigenvalues.cpp

```diff
@@ -0,0 +1,117 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "NumericalAlgorithms/LinearAlgebra/FindGeneralizedEigenvalues.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace {
+void test_find_generalized_eigenvalues(const Matrix& A, const Matrix& B,
+                                       const size_t eigenvector_size,
+                                       const size_t eigenvalue_size) {
+  // Set up vectors and a matrix to store the eigenvalues and eigenvectors
+  DataVector eigenvalues_real_part(eigenvalue_size, 0.0);
+  DataVector eigenvalues_im_part(eigenvalue_size, 0.0);
+  Matrix eigenvectors(eigenvector_size, eigenvector_size, 0.0);
+
+  // Get the generalized eigenvectors and eigenvalues
+  /// [generalized_eigenvalue_example]
```

Comment:
I personally don't see any value in a snippet that literally just shows someone how to call a function